setup.py

```diff
@@ -26,7 +26,7 @@
     license='LICENSE',
     zip_safe=False,
     packages=find_packages(exclude=['tests*']),
-    install_requires=['cloudify-plugins-common',
+    install_requires=['cloudify-common>=4.4',
```

Comment:
@ahmadiesa-abu I think this added here for a reason, so not sure if we should change this again to common

@EarthmanT @isaac-s

Reply:
@mabuaisha moving it back , it was merged from another branch by mistake

Reply:
Change is based on EOL https://cloudify.co/cloudify-lifecycle/ - looks like cloudify-plugins-common should retire. What do you think?

Reply:
@0lvin we decided to move it back as it should use the cloudify-plugins-common like it was before